src/Switches.h

```diff
@@ -94,7 +95,35 @@ class SwitchOnCycleDuration {
 		}
 		g_cycleStartTime = nextCycleStartTime;
 	}
+
+	inline void execute(int ifd, int buffer_size) {
```

Comment:
1. Better to use the original function with default arguments so you can call with and without input values.
2. Also add something line "NOT_IN_USE(x)" marco. Is there something like that in sockperf today?